telemetry/specs/INT.mdk

```diff
@@ -1407,14 +1657,80 @@ INT Metadata Header and Metadata Stack, followed by TCP header:
    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
 `
 
-## Example with INT in-between UDP header and UDP payload
+## Example with new UDP header and INT-MX inserted before TCP
+
+As before we consider a scenario where host1 sends a TCP packet to host2.
+The ToR switch of host1 (Switch1) acts as the INT source. It adds a new UDP header,
```

Comment:
nit: s/UDP header, INT-MX header/UDP header and a new INT-MX header

Reply:
Done